requirements/test.txt

```diff
@@ -17,3 +17,5 @@ cloudpickle>=1.3
 scikit-learn>0.22.1
 scikit-image>0.17.1
 nltk>=3.6
+scipy
```

Comment:
this is not only test, but all so all vision based

Reply:
scipy seems to be only used in tests for now, and torch-fidelity is optional for core library?